nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -127,6 +100,10 @@ target-version = "py38"
 [tool.ruff.lint]
 select = [
     "D",  # pydocstyle
+    "F", "E", "W",  # flake8
+    "PL",  # pylint
+    "S",  # bandit
+    "I",  # isort
```

Comment:
As mentioned in the internal ADR, please coordinate this, this is likely to cause PR conflicts so while yes we want this, it doesn't make sense imho to create a log jam of updates to be made.